apps/web/src/components/PoolCandidatesTable/SkillMatchDialog.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from "react";
+import { Experience, Maybe, Skill } from "@gc-digital-talent/graphql";
+import { Button, Dialog } from "@gc-digital-talent/ui";
+import { useIntl } from "react-intl";
+import SkillTree from "~/pages/Applications/ApplicationSkillsPage/components/SkillTree";
+
+interface SkillMatchDialogProps {
+  filteredSkills: Skill[];
+  experiences: Experience[];
+  skillsCount: Maybe<number>;
+}
+
+const SkillMatchDialog = ({
+  filteredSkills,
+  experiences,
+  skillsCount,
+}: SkillMatchDialogProps) => {
+  const intl = useIntl();
+
+  return (
+    <Dialog.Root>
+      <Dialog.Trigger>
+        <Button color="black" mode="inline">
+          {intl.formatMessage(
+            {
+              defaultMessage: "{skillsCount}/{filteredSkillsTotal}",
+              id: "npcY4+",
+              description:
+                "Title displayed on the candidate skill count column.",
+            },
+            { skillsCount, filteredSkillsTotal: filteredSkills.length },
+          )}
+        </Button>
+      </Dialog.Trigger>
+      <Dialog.Content>
+        <Dialog.Header>
+          {intl.formatMessage({
+            defaultMessage: "Skills matched in detail",
+            id: "qQZQj+",
+            description:
+              "Heading displayed on the candidate skill match dialog.",
+          })}
+        </Dialog.Header>
+        <Dialog.Body>
+          {filteredSkills.map((skill) => (
+            <SkillTree
+              key={skill.id}
+              skill={skill}
+              experiences={experiences}
+              showDisclaimer
+              disclaimerMessage={intl.formatMessage({
+                defaultMessage:
+                  "There are no experiences attached to this skill",
+                id: "pJqoQF",
+                description:
+                  "Disclaimer displayed in the skill tree on the skill match dialog.",
+              })}
+              hideConnectButton
+              hideEdit
+            />
+          ))}
+          <Dialog.Footer data-h2-justify-content="base(flex-start)">
+            <Dialog.Close>
+              <Button color="secondary" mode="inline">
+                {intl.formatMessage({
+                  defaultMessage: "Close",
+                  id: "GAYZpV",
+                  description:
+                    "Button text used to close an skills match dialog",
+                })}
+              </Button>
+            </Dialog.Close>
+          </Dialog.Footer>
+        </Dialog.Body>
+      </Dialog.Content>
+    </Dialog.Root>
+  );
+};
+
+export default SkillMatchDialog;
+
+export function skillMatchDialogAccessor(
+  filteredSkills: Skill[],
+  experiences: Experience[],
+  skillCount: Maybe<number>,
+) {
+  if (filteredSkills.length === 0) return <p>0/0</p>;
```

Comment:
@mnigh Should this be wrapped in an `intl.formatMessage`? I'm worried "0/0" is not universally correct.

Reply:
Not sure. If this format did not work in other languages, then how would `npcY4+` (`{skillsCount}/{filteredSkillsTotal}`)? Is there a language you are familiar with where this wouldn't work? A safer format would probably be `0 of 0` and `{skillsCount} of {filteredSkillsTotal}` though and then wrap that in `intl.formatMessage`.

Reply:
I am not aware of any, was just checking to see if you knew. I agree that `0 of 0` reads a bit better. 

---

Oh! Even more problematic is the buttons are not unique 😬 We may want to rethink the text for these 😓 

"Show 1 user skill of 4 selected skills" as an `aria-label` perhaps? 😿

Reply:
@yonikid15 I think that makes sense, but you might want to check with @Jerryescandon on this since the designs will need to be updated to reflect these changes.

Reply:
I talked to Jerbo and he approved, and the designs will get updated 👍. Also, I should get the translations sometime today (hopefully).